src/app/components/Hero/HeroInfo.tsx

```diff
@@ -1,38 +1,30 @@
 import { LogoFooter } from "@/app/icons/LogoFooter";
-import { PhoneButton } from "../Share/PhoneButton";
+import { HeroButtonBox } from "../Hero/HeroButtonBox"
 
 export const HeroInfo = () => {
   return (
     <>
```

Comment:
React fragment is unnecessary here, since the topmost element is a div that already surrounds the whole thing.